### ISSUE TITLE
Placement and translation of "Summary" 

### ISSUE THREAD

**Issue by killakalle** [theCrag/tca#94]
**What happened?**
Summar section is translated into Unique features and strengths. Why? The placement in the app is at the end

Example url(s) to reproduce the problem:

Open Cullera (Crag) in theCrag powered Topo Guru app! 
https://thecrag.topoguru.com/info/3385384056

Ideally also provide screenshots:
![Screenshot_20210408_231755_com topoguru](https://user-images.githubusercontent.com/30682796/114097881-f070d000-98c0-11eb-879c-3db19c594d39.jpg)
![Screenshot_20210408_231849_com android chrome](https://user-images.githubusercontent.com/30682796/114097896-f49ced80-98c0-11eb-8d9d-2224115a687a.jpg)

**What you expected:**

* [ ] Same title as in theCrag
* [ ] Placed before/above Description
* [ ] nice-to-have: use summary text somehow while browsing crag/sector list
* [ ] "Description" heading only once - not twice

**Comment by rouletout** [comment]
Will do points 1, 2 and 4

**Comment by bkucsera** [comment]
Edited:
1. "Unique Features and Strengths" renamed to Summary. 
2. Summary moved to the top.
3. we can put it to the Enhancement list
4. done

**Comment by bkucsera** [comment]
Please check and close.
